src/style.css

```diff
@@ -1 +1,81 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  margin-left: 0;
+}
+
+.header {
+  margin: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  margin: 0;
+  display: flex;
+  padding: 0;
```

Comment:
Reset all margin and padding which u need like we did it on the lesson

```
ul,
p,
... {
  mar: 0
  pad: 0
}
```

Reply:
Fix Ihor's comment